providers/microsoft/mssql/pyproject.toml

```
@@ -59,6 +60,8 @@ dependencies = [
     "apache-airflow>=2.10.0",
     "apache-airflow-providers-common-sql>=1.23.0",
     "pymssql>=2.3.5",
+    "methodtools>=0.4.7",
+    "methodtools>=0.4.7",
```

Uuups, now you are adding this two times?
```suggestion
    "methodtools>=0.4.7",
```

---

It's actually removed now :)

---

Somehow you made it back again

---

> Somehow you made it back again 

🪃

---

Fixed.